README.md

```diff
@@ -69,11 +71,28 @@ You can also choose one of the presets provided by the ROS2 Gem to create a LiDA
 the **RUNTIME** requirements.*
 
 ## Setup
+
 1. **Clone the Gem's repository.**
     ```bash
     git clone https://github.com/RobotecAI/o3de-rgl-gem.git
     ```
-2. **Register the Gem.** \
+2. **Switch to the correct branch.** \
+   Depending on which version of O3DE and ROS2 gem you use this step may differ.
+   Below is a universal table of supported build combinations:
+
+   | o3de-rgl-gem | [o3de/o3de-extras](https://github.com/o3de/o3de-extras) | [Robotecai/o3de-extras](https://github.com/robotecai/o3de-extras) | [o3de](https://github.com/o3de/o3de)                |
+   |--------------|---------------------------------------------------------|-------------------------------------------------------------------|-----------------------------------------------------|
+   | development  | development                                             | N/A                                                               | development (or latest stabilization if compatible) |
+   | development  | N/A                                                     | rgl/stable-dev                                                    | latest point release                                |
+   | main         | latest point release                                    | N/A                                                               | latest point release                                |
```

Comment:
I would consider using tag / specific version numbers for point releases - it will require additional effort with new release. but it will be easier to read. rgl/stable-dev should probably get some tag too in the near future.